x/adminmodule/keeper/querier.go

```diff
@@ -0,0 +1,49 @@
+package keeper
+
+import (
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/neutron-org/neutron/x/adminmodule/types"
+)
+
+// NewQuerier creates a new adminmodule Querier instance
+func NewQuerier(keeper Keeper, legacyQuerierCdc *codec.LegacyAmino) sdk.Querier {
```

Comment:
i don't think this is the way to go in sdk 0.45; please check the query implementations in our modules